applications/nrf5340_audio/src/modules/sd_card.c

```diff
@@ -127,12 +127,12 @@ int sd_card_open_write_close(char const *const filename, char const *const data,
 	struct fs_file_t f_entry;
 	char abs_path_name[PATH_MAX_LEN + 1] = SD_ROOT_PATH;
 
-	if (k_sem_count_get(&m_sem_sd_oper_ongoing) <= 0) {
-		LOG_ERR("SD operation ongoing");
-		return -EPERM;
-	}
+	ret = k_sem_take(&m_sem_sd_oper_ongoing, K_MSEC(K_SEM_OPER_TIMEOUT_MS));
 
```

Comment:
We usually do not have newlines between the function and the if (ret) check. Valid several places in this file